Neos.ContentRepository.Core/Classes/ContentRepositoryReadModel.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core;
```

Comment:
Im not sure about the namespace, as its first level.

There was a time where only the `ContentRepository` was in the first level and IMO that makes sense since its api and should be used. All the other internal stuff also the `CommandHandlingDependencies` should reside in another namespace i think.

`CommandHandlingDependencies` -> `Neos\ContentRepository\Core\CommandHandler`

but the question is where to put this `ContentRepositoryReadModel`?
Maybe in `Projection`?

Reply:
I agree. This should be moved!

Reply:
Can also be adjusted when finalising everything in https://github.com/neos/neos-development-collection/pull/5272